src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -249,9 +256,42 @@ public AzureAppConfigurationOptions Connect(Uri endpoint, TokenCredential creden
                 throw new ArgumentNullException(nameof(credential));
             }
 
+            return Connect(new List<Uri>() { endpoint }, credential);
+        }
+
+        /// <summary>
+        /// Connect the provider to Azure App Configuration and their replicas using list of endpoints and token credential.
+        /// </summary>
+        /// <param name="endpoints">The list of endpoints of the Azure App Configuration and it's replicas to connect to.</param>
```

Comment:
NIT:
the Azure App Configuration -> an Azure App Configuration store
it's -> its